### PR TITLE
Update approvers list

### PR DESCRIPTION
**Pull request by svetsa-rh**
@paigerube14 Can you ptal?

### CODE REVIEW

File: ci-operator/step-registry/openshift-qe/installer/OWNERS [hunk 1]
```diff
@@ -3,6 +3,7 @@ approvers:
 - paigerube14
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/installer/OWNERS:3]: If this folder OWNERS file is the same as the file in the openshift-qe main folder think we should have a symbolic link here so we dont need to update every sub folder every time we have an edit to the owners

Example: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/cluster-density-v2/OWNERS

Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/installer/OWNERS]: @paigerube14 Updated, thanks!

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/OWNERS [hunk 2]
```diff
@@ -9,6 +9,7 @@ approvers:
 - mffiedler
 - paigerube14
 - qiliredhat
+- svetsa-rh
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/OWNERS:12]: this file should probably also use the symbolic link

Comment (svetsa-rh) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/OWNERS]: Updated.

**Pull request merged**